packages/core/src/directives/template-map.ts

```diff
@@ -68,20 +72,18 @@ const setMapped = (element: Element, name: string, value: unknown, oldValue: unk
     case MAP_TYPE.PROPERTY:
       if (value === undefined) {
         delete (element as VariableElement)[name];
-      }
-      else {
+      } else {
         (element as VariableElement)[name] = value;
       }
       break;
     case MAP_TYPE.ATTRIBUTE:
     default:
-      if (value === null || value === undefined || value === false) { // remove with undefined to comply with property
+      if (value === null || value === undefined || value === false) {
+        // remove with undefined to comply with property
```

Comment:
This comment describes `if (value === null || value === undefined || value === false) {`, the code proceeding it. With this formatting, the comment would ostensibly describing its following line instead.

Reply:
Agree. I'm going move comment to just above the `if`

Reply:
A comment might be describing something above rather than below as a result of Prettier formatting. As there is no code behavior change, marking as resolved.